src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Confirm.tsx

```diff
@@ -99,14 +106,37 @@ export const Confirm: React.FC<StepComponentProps & { accountId: AccountId | und
   )
 
   const handleDeposit = useCallback(async () => {
-    if (!dispatch || !state || !accountAddress || !assetReference || !walletState.wallet) return
+    if (
+      !dispatch ||
+      !state ||
+      !accountAddress ||
+      !assetReference ||
+      !walletState.wallet ||
+      !asset ||
+      !foxFarmingOpportunity
+    )
+      return
     try {
       dispatch({ type: FoxFarmingDepositActionType.SET_LOADING, payload: true })
       const txid = await stake(state.deposit.cryptoAmount)
       if (!txid) throw new Error('Transaction failed')
       dispatch({ type: FoxFarmingDepositActionType.SET_TXID, payload: txid })
       onOngoingFarmingTxIdChange(txid, contractAddress)
       onNext(DefiStep.Status)
+      trackOpportunityEvent(
+        MixPanelEvents.DepositConfirm,
+        {
+          opportunity: foxFarmingOpportunity,
+          fiatAmounts: [state.deposit.fiatAmount],
+          cryptoAmounts: [
+            {
+              assetId: asset.assetId,
+              amountCryptoHuman: state.deposit.cryptoAmount,
+            },
+          ],
```

Comment:
Do we also want to track crypto / fiat underlyings ? It's data we can get fairly easily with our helpers, so might as well